diff --git a/arch/riscv/boot/dts/starfive/dubhe_fpga_dual.dts b/arch/riscv/boot/dts/starfive/dubhe_fpga_dual.dts
index 86d69ea6c73b..527e69bb0798 100644
--- a/arch/riscv/boot/dts/starfive/dubhe_fpga_dual.dts
+++ b/arch/riscv/boot/dts/starfive/dubhe_fpga_dual.dts
@@ -11,7 +11,7 @@ aliases {
 	};
 
 	chosen {
-		bootargs = "console=ttySIF0,115200 earlycon=sbi";
+		bootargs = "console=ttySIF0,115200 earlycon=sbi root=/dev/mmcblk0p2 rw rootfstype=ext4 rootwait";
 	};
 
 	cpus {
