diff --git a/arch/riscv/boot/dts/starfive/dubhe_fpga.dts b/arch/riscv/boot/dts/starfive/dubhe_fpga.dts
index 89098aab24d9..0d43a18bd446 100644
--- a/arch/riscv/boot/dts/starfive/dubhe_fpga.dts
+++ b/arch/riscv/boot/dts/starfive/dubhe_fpga.dts
@@ -12,7 +12,7 @@ aliases {
 	};
 
 	chosen {
-		bootargs = "console=ttySIF0,115200 earlycon=sbi root=/dev/nfs nfsroot=192.168.153.221:/home/nfsroot,rw,tcp,vers=3 ip=192.168.152.100:192.168.153.221:192.168.152.1:255.255.255.0::eth0:off";
+		bootargs = "console=ttySIF0,115200 earlycon=sbi root=/dev/nfs rw nfsroot=192.168.1.1:/filepath,rw,tcp,vers=3 ip=:::255.255.255.0::eth0:dhcp rootfstype=ext4 rootwait";
 	};
 
 	cpus {
