From d113ead74f33f47a2704c717c56304f191d11239 Mon Sep 17 00:00:00 2001
From: "yilun.xie" <yilun.xie@starfivetech.com>
Date: Wed, 17 Jul 2024 10:26:29 +0800
Subject: [PATCH 4/5] RISCV: Fix multilib config for baremetal

---
 gcc/config/riscv/t-elf-multilib | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/gcc/config/riscv/t-elf-multilib b/gcc/config/riscv/t-elf-multilib
index 2d17b2e026a..34f023cec78 100644
--- a/gcc/config/riscv/t-elf-multilib
+++ b/gcc/config/riscv/t-elf-multilib
@@ -1,11 +1,9 @@
-#  ./gcc/config/riscv/multilib-generator rv64imafdc_zicsr_zifencei-lp64d-- rv64imafdc_zicsr_zifencei_zba_zbb_zbc_zbs-lp64d-- rv64imafdcv_zicsr_zifencei_zvl256b-lp64d-- rv64imafdcv_zicsr_zifencei_zba_zbb_zbc_zbs_zvl256b-lp64d--
-MULTILIB_OPTIONS = march=rv64imafdc_zicsr_zifencei/march=rv64imafdc_zicsr_zifencei_zba_zbb_zbc_zbs/march=rv64imafdcv_zicsr_zifencei_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b/march=rv64imafdcv_zicsr_zifencei_zba_zbb_zbc_zbs_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b mabi=lp64d 
+#  ./gcc/config/riscv/multilib-generator rv64imafdc_zicsr_zifencei-lp64d-- rv64imafdcv_zicsr_zifencei_zvl256b-lp64d-- rv64imafdcv_zicsr_zifencei_zba_zbb_zbs_zvl256b-lp64d--
+MULTILIB_OPTIONS = march=rv64imafdc_zicsr_zifencei/march=rv64imafdcv_zicsr_zifencei_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b/march=rv64imafdcv_zicsr_zifencei_zba_zbb_zbs_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b mabi=lp64d 
 MULTILIB_DIRNAMES = rv64imafdc_zicsr_zifencei \
-rv64imafdc_zicsr_zifencei_zba_zbb_zbc_zbs \
 rv64imafdcv_zicsr_zifencei_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b \
-rv64imafdcv_zicsr_zifencei_zba_zbb_zbc_zbs_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b lp64d
+rv64imafdcv_zicsr_zifencei_zba_zbb_zbs_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b lp64d
 MULTILIB_REQUIRED = march=rv64imafdc_zicsr_zifencei/mabi=lp64d \
-march=rv64imafdc_zicsr_zifencei_zba_zbb_zbc_zbs/mabi=lp64d \
 march=rv64imafdcv_zicsr_zifencei_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b/mabi=lp64d \
-march=rv64imafdcv_zicsr_zifencei_zba_zbb_zbc_zbs_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b/mabi=lp64d
+march=rv64imafdcv_zicsr_zifencei_zba_zbb_zbs_zve32f_zve32x_zve64d_zve64f_zve64x_zvl128b_zvl256b_zvl32b_zvl64b/mabi=lp64d
 MULTILIB_REUSE =
-- 
2.25.1

