From 93b48d269ad1ce912f72b396f6b1e0442a24b425 Mon Sep 17 00:00:00 2001
From: GCC Administrator <gccadmin@gcc.gnu.org>
Date: Mon, 5 Dec 2022 00:21:42 +0000
Subject: [PATCH 260/547] Daily bump.

---
 gcc/DATESTAMP | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/DATESTAMP b/gcc/DATESTAMP
index 1638a1245b8..fac0e17b9fa 100644
--- a/gcc/DATESTAMP
+++ b/gcc/DATESTAMP
@@ -1 +1 @@
-20221204
+20221205
-- 
2.25.1

