From a3fbfc1027e9edcd14bb290b5702504d80d9e8fe Mon Sep 17 00:00:00 2001
From: GCC Administrator <gccadmin@gcc.gnu.org>
Date: Tue, 27 Dec 2022 00:20:21 +0000
Subject: [PATCH 317/547] Daily bump.

---
 gcc/DATESTAMP | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/DATESTAMP b/gcc/DATESTAMP
index 6c91a6094a4..154fbb15dde 100644
--- a/gcc/DATESTAMP
+++ b/gcc/DATESTAMP
@@ -1 +1 @@
-20221226
+20221227
-- 
2.25.1

