From f816ad1481c4054ed011419663b19b2720f4471f Mon Sep 17 00:00:00 2001
From: GCC Administrator <gccadmin@gcc.gnu.org>
Date: Sat, 31 Dec 2022 00:20:24 +0000
Subject: [PATCH 322/547] Daily bump.

---
 gcc/DATESTAMP | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/DATESTAMP b/gcc/DATESTAMP
index 13c8be464ab..e1dea3478c0 100644
--- a/gcc/DATESTAMP
+++ b/gcc/DATESTAMP
@@ -1 +1 @@
-20221230
+20221231
-- 
2.25.1

