From 56314aa33a46224bbe070f8cddb7dda2a5aea747 Mon Sep 17 00:00:00 2001
From: GCC Administrator <gccadmin@gcc.gnu.org>
Date: Tue, 10 Jan 2023 00:22:35 +0000
Subject: [PATCH 338/547] Daily bump.

---
 gcc/DATESTAMP       | 2 +-
 libcpp/po/ChangeLog | 4 ++++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/gcc/DATESTAMP b/gcc/DATESTAMP
index 6a739acb152..3437f90b0b3 100644
--- a/gcc/DATESTAMP
+++ b/gcc/DATESTAMP
@@ -1 +1 @@
-20230109
+20230110
diff --git a/libcpp/po/ChangeLog b/libcpp/po/ChangeLog
index ffe71140a6e..4c0d40272e7 100644
--- a/libcpp/po/ChangeLog
+++ b/libcpp/po/ChangeLog
@@ -1,3 +1,7 @@
+2023-01-09  Joseph Myers  <joseph@codesourcery.com>
+
+	* eo.po: Update.
+
 2022-10-10  Joseph Myers  <joseph@codesourcery.com>
 
 	* ro.po: New.
-- 
2.25.1

