From 3ae7a822456bc538d4cefaa3a22fe56d43640a04 Mon Sep 17 00:00:00 2001
From: GCC Administrator <gccadmin@gcc.gnu.org>
Date: Sat, 21 Jan 2023 00:20:54 +0000
Subject: [PATCH 356/547] Daily bump.

---
 gcc/DATESTAMP | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/DATESTAMP b/gcc/DATESTAMP
index c8519a7a377..583ce61ca83 100644
--- a/gcc/DATESTAMP
+++ b/gcc/DATESTAMP
@@ -1 +1 @@
-20230120
+20230121
-- 
2.25.1

