From b9f5d34ddc17e990e2d21929b25999d56493c3e0 Mon Sep 17 00:00:00 2001
From: GCC Administrator <gccadmin@gcc.gnu.org>
Date: Mon, 13 Feb 2023 00:21:15 +0000
Subject: [PATCH 469/547] Daily bump.

---
 gcc/DATESTAMP | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/DATESTAMP b/gcc/DATESTAMP
index d696f4e8b1a..d3e275017e1 100644
--- a/gcc/DATESTAMP
+++ b/gcc/DATESTAMP
@@ -1 +1 @@
-20230212
+20230213
-- 
2.25.1

