From 652ad832889d6a5516defe42b3ab8ec6d9b43eb8 Mon Sep 17 00:00:00 2001
From: Tien Hock Loh <tienhock.loh@starfivetech.com>
Date: Thu, 5 Aug 2021 05:08:09 +0000
Subject: [PATCH 1/1] Fix compilation issue when building multilib

Signed-off-by: Tien Hock Loh <tienhock.loh@starfivetech.com>
---
 gcc/common/common-targhooks.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/common/common-targhooks.h b/gcc/common/common-targhooks.h
index 66a54c2cb9c..3af6446db03 100644
--- a/gcc/common/common-targhooks.h
+++ b/gcc/common/common-targhooks.h
@@ -32,4 +32,5 @@ extern vec<const char *> default_get_valid_option_values (int, const char *);
 
 extern const struct default_options empty_optimization_table[];
 
+extern const char *default_compute_multilib(const struct switchstr *, int, const char *multilib, const char *, const char *, const char *, const char *, const char *);
 #endif
-- 
2.25.1

