From 0547b0b41692e597730385d76f31699eb0fc72bc Mon Sep 17 00:00:00 2001
From: Kito Cheng <kito.cheng@sifive.com>
Date: Wed, 30 Dec 2020 11:34:27 +0800
Subject: [PATCH 50/60] RISC-V: Remove duplicated define for zfh

---
 gcc/config/riscv/riscv-c.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/gcc/config/riscv/riscv-c.c b/gcc/config/riscv/riscv-c.c
index 771ab343d27..6e3e08c46f4 100644
--- a/gcc/config/riscv/riscv-c.c
+++ b/gcc/config/riscv/riscv-c.c
@@ -57,9 +57,6 @@ riscv_cpu_cpp_builtins (cpp_reader *pfile)
   if (TARGET_HARD_FLOAT)
     builtin_define_with_int_value ("__riscv_flen", UNITS_PER_FP_REG * 8);
 
-  if (TARGET_FP16)
-    builtin_define ("__riscv_zfh");
-
   if (TARGET_HARD_FLOAT && TARGET_FDIV)
     {
       builtin_define ("__riscv_fdiv");
-- 
2.25.1

