From 83ecf68125f72c5a112e57274cb1d106ced6a47e Mon Sep 17 00:00:00 2001
From: Kito Cheng <kito.cheng@sifive.com>
Date: Mon, 18 Jan 2021 13:47:56 +0800
Subject: [PATCH 58/60] Add more comment for riscv_check_other_cond

---
 gcc/common/config/riscv/riscv-common.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gcc/common/config/riscv/riscv-common.c b/gcc/common/config/riscv/riscv-common.c
index 27bf9828325..11180307064 100644
--- a/gcc/common/config/riscv/riscv-common.c
+++ b/gcc/common/config/riscv/riscv-common.c
@@ -1320,6 +1320,9 @@ riscv_check_other_cond (
       p++;
     }
 
+  /* 100 is magic number, it's just used for make sure this multi-lib has
+     higher priority if we found any some option is listed in the option check
+     list. */
   return match_score + ok_count * 100;
 }
 
-- 
2.25.1

