From ede43bd5bc429624f75808ca1060be15b05c2b23 Mon Sep 17 00:00:00 2001
From: "demin.han" <demin.han@starfivetech.com>
Date: Wed, 15 Mar 2023 15:22:49 +0800
Upstream-Status: Inappropriate [riscv-specific]
Subject: [PATCH 1/1] default enable align-functions=16 and align-loops=8

---
 gcc/config/riscv/riscv.cc | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/gcc/config/riscv/riscv.cc b/gcc/config/riscv/riscv.cc
index d3656a7a430..bd2b71230d3 100644
--- a/gcc/config/riscv/riscv.cc
+++ b/gcc/config/riscv/riscv.cc
@@ -10628,6 +10628,19 @@ riscv_override_options_internal (struct gcc_options *opts)
   if (opts->x_riscv_branch_cost == 0)
     opts->x_riscv_branch_cost = tune_param->branch_cost;
 
+  if (!opts->x_optimize_size)
+    {
+  /* If the user hasn't specified the loop align, use '8' by default.  */
+      if (opts->x_flag_align_loops && !opts->x_str_align_loops)
+	opts->x_str_align_loops = "8";
+  /* If the user hasn't specified the jump align, use '8' by default.  */
+      if (opts->x_flag_align_jumps && !opts->x_str_align_jumps)
+	opts->x_str_align_jumps = "0";
+  /* If the user hasn't specified the jump align, use '8' by default.  */
+      if (opts->x_flag_align_functions && !opts->x_str_align_functions)
+	opts->x_str_align_functions = "16";
+    }
+
   /* FIXME: We don't allow TARGET_MIN_VLEN > 4096 since the datatypes of
      both GET_MODE_SIZE and GET_MODE_BITSIZE are poly_uint16.
 
-- 
2.43.0

