From 75e084199a258a916172f01c9e061fb3ee6cba54 Mon Sep 17 00:00:00 2001
From: "yilun.xie" <yilun.xie@starfivetech.com>
Date: Mon, 19 Aug 2024 15:44:00 +0800
Subject: [PATCH 7/7] add starfive dubhe fusion patterns

---
 gcc/config/riscv/riscv.cc | 250 +++++++++++++++++++++++++++++++++++++-
 1 file changed, 248 insertions(+), 2 deletions(-)

diff --git a/gcc/config/riscv/riscv.cc b/gcc/config/riscv/riscv.cc
index ca6b3b2fe3b..6d3ee246a4e 100644
--- a/gcc/config/riscv/riscv.cc
+++ b/gcc/config/riscv/riscv.cc
@@ -270,6 +270,17 @@ enum riscv_fusion_pairs
   RISCV_FUSE_AUIPC_LD = (1 << 7),
   RISCV_FUSE_LDPREINCREMENT = (1 << 8),
   RISCV_FUSE_ALIGNED_STD = (1 << 9),
+  RISCV_FUSE_ADDUW_LOAD = (1 << 10),
+  RISCV_FUSE_SH1ADD_LH = (1 << 11),
+  RISCV_FUSE_SH1ADDUW_LH = (1 << 12),
+  RISCV_FUSE_SH2ADD_LW = (1 << 13),
+  RISCV_FUSE_SH2ADDUW_LW = (1 << 14),
+  RISCV_FUSE_SH3ADD_LD = (1 << 15),
+  RISCV_FUSE_SH3ADDUW_LD = (1 << 16),
+  RISCV_FUSE_SLLI_ADD = (1 << 17),
+  RISCV_FUSE_SLLIUW_ADD = (1 << 18),
+  RISCV_FUSE_ADDI_BRANCH = (1 << 19),
+  RISCV_FUSE_ADDI_JALR = (1 << 20),
 };
 
 /* Costs of various operations on the different architectures.  */
@@ -528,7 +539,14 @@ static const struct riscv_tune_param starfive_dubhe_tune_info = {
   8,						/* fmv_cost */
   true,						/* slow_unaligned_access */
   false,					/* use_divmod_expansion */
-  RISCV_FUSE_NOTHING,                           /* fusible_ops */
+  (RISCV_FUSE_LDINDEXED | RISCV_FUSE_ADDUW_LOAD |
+   RISCV_FUSE_SH1ADD_LH | RISCV_FUSE_SH1ADDUW_LH |
+   RISCV_FUSE_SH2ADD_LW | RISCV_FUSE_SH2ADDUW_LW |
+   RISCV_FUSE_SH3ADD_LD | RISCV_FUSE_SH3ADDUW_LD |
+   RISCV_FUSE_LUI_LD | RISCV_FUSE_LUI_ADDI |
+   RISCV_FUSE_SLLI_ADD | RISCV_FUSE_SLLIUW_ADD |
+   RISCV_FUSE_ADDI_BRANCH | RISCV_FUSE_ADDI_JALR |
+   RISCV_FUSE_AUIPC_ADDI),                      /* fusible_ops */
   NULL,						/* vector cost */
 };
 
@@ -8734,7 +8752,7 @@ riscv_macro_fusion_pair_p (rtx_insn *prev, rtx_insn *curr)
 	return true;
     }
 
-    if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_LDPREINCREMENT))
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_LDPREINCREMENT))
     {
       /* We are trying to match the following:
 	   prev (add) == (set (reg:DI rS)
@@ -8873,6 +8891,234 @@ riscv_macro_fusion_pair_p (rtx_insn *prev, rtx_insn *curr)
 	}
     }
 
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_ADDUW_LOAD))
+    {
+      /* We are trying to match the following:
+	   prev (add.uw) == (set (reg:DI rD)
+			      (plus:DI (zero_extend:DI (reg:SI rS1)) (reg:DI rS2)))
+	   curr (ld)  == (set (reg:DI rD)
+			      (mem:DI (reg:DI rD))) */
+
+      if (MEM_P (SET_SRC (curr_set))
+	  && REG_P (XEXP (SET_SRC (curr_set), 0))
+	  && REGNO (XEXP (SET_SRC (curr_set), 0)) == REGNO (SET_DEST (prev_set))
+	  && GET_CODE (SET_SRC (prev_set)) == PLUS
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == ZERO_EXTEND
+	  && REG_P (XEXP (XEXP (SET_SRC (prev_set), 0), 0))
+	  && REG_P (XEXP (SET_SRC (prev_set), 1)))
+	return true;
+
+      /* We are trying to match the following:
+	   prev (add.uw) == (set (reg:DI rD)
+			      (plus:DI (zero_extend:DI (reg:SI rS1)) (reg:DI rS2)))
+	   curr (lw)  == (set (any_extend:DI (mem:SUBX (reg:DI rD)))) */
+
+      if ((GET_CODE (SET_SRC (curr_set)) == SIGN_EXTEND
+	   || (GET_CODE (SET_SRC (curr_set)) == ZERO_EXTEND))
+	  && MEM_P (XEXP (SET_SRC (curr_set), 0))
+	  && REG_P (XEXP (XEXP (SET_SRC (curr_set), 0), 0))
+	  && REGNO (XEXP (XEXP (SET_SRC (curr_set), 0), 0)) == REGNO (SET_DEST (prev_set))
+	  && GET_CODE (SET_SRC (prev_set)) == PLUS
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == ZERO_EXTEND
+	  && REG_P (XEXP (XEXP (SET_SRC (prev_set), 0), 0))
+	  && REG_P (XEXP (SET_SRC (prev_set), 1)))
+	return true;
+    }
+
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_SH3ADD_LD))
+    {
+      /* We are trying to match the following:
+	   prev (sh3add) == (set (reg:DI rD)
+			          (plus:DI (ashift:DI (reg:DI rS1) (const_int 3))
+                         (reg:DI rS2)))
+	   curr (ld)     == (set (reg:DI rD)
+                (mem:DI (reg:DI rD))) */
+
+      if (MEM_P (SET_SRC (curr_set))
+	  && REG_P (XEXP (SET_SRC (curr_set), 0))
+    && REGNO (XEXP (SET_SRC (curr_set), 0)) == REGNO (SET_DEST (prev_set))
+    && GET_CODE (SET_SRC (prev_set)) == PLUS
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == ASHIFT
+    && REG_P (XEXP (XEXP (SET_SRC (prev_set), 0), 0))
+    && INTVAL (XEXP (XEXP (SET_SRC (prev_set), 0), 1)) == 3
+    && REG_P (XEXP (SET_SRC (prev_set), 1)))
+  return true;
+    }
+
+  if (simple_sets_p && (riscv_fusion_enabled_p (RISCV_FUSE_SH1ADD_LH)
+    || riscv_fusion_enabled_p (RISCV_FUSE_SH2ADD_LW)))
+    {
+      /* We are trying to match the following:
+	   prev (sh1add) == (set (reg:DI rD)
+			          (plus:DI (ashift:DI (reg:DI rS1) (const_int 1))
+                         (reg:DI rS2)))
+	   curr (lh)     == (set (reg:DI rD)
+                (sign_extend/zero_extend:DI (mem:SUBX (reg:HI rD))))
+
+	   prev (sh2add) == (set (reg:DI rD)
+			          (plus:DI (ashift:DI (reg:DI rS1) (const_int 2))
+                         (reg:DI rS2)))
+	   curr (lw)     == (set (reg:DI rD)
+                (sign_extend/zero_extend:DI (mem:SUBX (reg:SI rD)))) */
+
+      if ((GET_CODE (SET_SRC (curr_set)) == SIGN_EXTEND
+     || GET_CODE (SET_SRC (curr_set)) == ZERO_EXTEND)
+    && MEM_P (XEXP (SET_SRC (curr_set), 0))
+    && REG_P (XEXP (XEXP (SET_SRC (curr_set), 0), 0))
+    && REGNO (XEXP (XEXP (SET_SRC (curr_set), 0), 0)) == REGNO (SET_DEST (prev_set))
+    && GET_CODE (SET_SRC (prev_set)) == PLUS
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == ASHIFT
+    && REG_P (XEXP (XEXP (SET_SRC (prev_set), 0), 0))
+    && INTVAL (XEXP (XEXP (SET_SRC (prev_set), 0), 1)) >= 1
+    && INTVAL (XEXP (XEXP (SET_SRC (prev_set), 0), 1)) <= 2
+    && REG_P (XEXP (SET_SRC (prev_set), 1)))
+  return true;
+    }
+
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_SH3ADDUW_LD))
+    {
+      /* We are trying to match the following:
+	   prev (sh3add.uw) == (set (reg:DI rD1)
+			            (plus:DI (and:DI (ashift:DI (reg:DI rS1) (const_int 3)) (const_int))
+                                   (reg:DI rS2)))
+	   curr (ld)        == (set (reg:DI rD1)
+                  (mem:DI (reg:DI rD1))) */
+
+      if (MEM_P (SET_SRC (curr_set))
+    && REG_P (XEXP (SET_SRC (curr_set), 0))
+    && REGNO (XEXP (SET_SRC (curr_set), 0)) == REGNO (SET_DEST (prev_set))
+    && GET_CODE (SET_SRC (prev_set)) == PLUS
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == AND
+    && GET_CODE (XEXP (XEXP (SET_SRC (prev_set), 0), 0)) == ASHIFT
+    && REG_P (XEXP (XEXP (XEXP (SET_SRC (prev_set), 0), 0), 0))
+    && INTVAL (XEXP (XEXP (XEXP (SET_SRC (prev_set), 0), 0), 1)) == 3
+    && REG_P (XEXP (SET_SRC (prev_set), 1)))
+  return true;
+    }
+
+  if (simple_sets_p && (riscv_fusion_enabled_p (RISCV_FUSE_SH1ADDUW_LH)
+    || riscv_fusion_enabled_p (RISCV_FUSE_SH2ADDUW_LW)))
+    {
+      /* We are trying to match the following:
+	   prev (sh1add.uw) == (set (reg:DI rD1)
+			            (plus:DI (and:DI (ashift:DI (reg:DI rS1) (const_int 1)) (const_int))
+                                   (reg:DI rS2)))
+	   curr (lh)        == (set (reg:DI rD1)
+                  (sign_extend/zero_extend:DI (mem:SUBX (reg:HI rD1))))
+
+     prev (sh2add.uw) == (set (reg:DI rD1)
+			            (plus:DI (and:DI (ashift:DI (reg:DI rS1) (const_int 1)) (const_int))
+                                   (reg:DI rS2)))
+	   curr (lw)        == (set (reg:DI rD1)
+                  (sign_extend/zero_extend:DI (mem:SUBX (reg:SI rD1)))) */
+      
+      if ((GET_CODE (SET_SRC (curr_set)) == SIGN_EXTEND
+     || GET_CODE (SET_SRC (curr_set)) == ZERO_EXTEND)
+    && MEM_P (XEXP (SET_SRC (curr_set), 0))
+    && REG_P (XEXP (XEXP (SET_SRC (curr_set), 0), 0))
+    && REGNO (XEXP (XEXP (SET_SRC (curr_set), 0), 0)) == REGNO (SET_DEST (prev_set))
+    && GET_CODE (SET_SRC (prev_set)) == PLUS
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == AND
+    && GET_CODE (XEXP (XEXP (SET_SRC (prev_set), 0), 0)) == ASHIFT
+    && REG_P (XEXP (XEXP (XEXP (SET_SRC (prev_set), 0), 0), 0))
+    && INTVAL (XEXP (XEXP (XEXP (SET_SRC (prev_set), 0), 0), 1)) >= 1
+    && INTVAL (XEXP (XEXP (XEXP (SET_SRC (prev_set), 0), 0), 1)) <= 2
+    && REG_P (XEXP (SET_SRC (prev_set), 1)))
+  return true;
+    }
+
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_SLLI_ADD))
+    {
+      /* We are trying to match the following:
+	   prev (slli) == (set (reg:DI rD)
+			        (ashift:X (reg:X rS1) (const_int imm)))
+	   curr (add)  == (set (reg:DI rD)
+			        (plus:DI (reg:DI rD) (reg:DI rS2))) */
+
+      if (GET_CODE (SET_SRC (curr_set)) == PLUS
+    && REG_P (XEXP (SET_SRC (curr_set), 0))
+    && REG_P (XEXP (SET_SRC (curr_set), 1))
+    && (REGNO (XEXP (SET_SRC (curr_set), 0)) == REGNO (SET_DEST (prev_set))
+     || REGNO (XEXP (SET_SRC (curr_set), 1)) == REGNO (SET_DEST (prev_set)))
+    && REG_P (SET_DEST (curr_set))
+    && REG_P (SET_DEST (prev_set))
+    && REGNO (SET_DEST (curr_set)) == REGNO (SET_DEST (prev_set))
+	  && GET_CODE (SET_SRC (prev_set)) == ASHIFT
+    && CONST_INT_P (XEXP (SET_SRC (prev_set), 1)))
+  return true;
+    }
+
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_SLLIUW_ADD))
+    {
+      /* We are trying to match the following:
+	   prev (slli.uw) == (set (reg:DI rD)
+	              (and:DI (ashift:DI (reg:DI rS1) (const_int imm))
+		                    (const_int)))
+	   curr (add)     == (set (reg:DI rD)
+			          (plus:DI (reg:DI rD) (reg:DI rS2))) */
+
+      if (GET_CODE (SET_SRC (curr_set)) == PLUS
+    && REG_P (XEXP (SET_SRC (curr_set), 0))
+    && REG_P (XEXP (SET_SRC (curr_set), 1))
+    && (REGNO (XEXP (SET_SRC (curr_set), 0)) == REGNO (SET_DEST (prev_set))
+     || REGNO (XEXP (SET_SRC (curr_set), 1)) == REGNO (SET_DEST (prev_set)))
+    && REG_P (SET_DEST (prev_set))
+    && REG_P (SET_DEST (curr_set))
+    && REGNO (SET_DEST (curr_set)) == REGNO (SET_DEST (prev_set))
+	  && GET_CODE (SET_SRC (prev_set)) == AND
+    && GET_CODE (XEXP (SET_SRC (prev_set), 0)) == ASHIFT
+    && CONST_INT_P (XEXP (XEXP (SET_SRC (prev_set), 0), 1)))
+  return true;
+    }
+
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_ADDI_BRANCH))
+    {
+      /* We are trying to match the following:
+	   prev (addi) == (set (reg:DI rD)
+			        (plus:DI (reg:DI X0) (const_int IMM12)))
+	   curr (BEQ/BNE/BLT/BGE)  == (set (pc)
+	            (if_then_else
+              (eq/ne/lt/ge:(reg:DI rD)(reg:DI rS1))
+	            (label_ref (const_int IMM12))
+	            (pc))) */
+
+      if (GET_CODE (SET_SRC (curr_set)) == IF_THEN_ELSE
+    && (GET_CODE (XEXP (SET_SRC (curr_set), 0)) == EQ
+     || GET_CODE (XEXP (SET_SRC (curr_set), 0)) == NE
+     || GET_CODE (XEXP (SET_SRC (curr_set), 0)) == LT
+     || GET_CODE (XEXP (SET_SRC (curr_set), 0)) == LTU
+     || GET_CODE (XEXP (SET_SRC (curr_set), 0)) == GE
+     || GET_CODE (XEXP (SET_SRC (curr_set), 0)) == GEU)
+    && REG_P (XEXP (XEXP (SET_SRC (curr_set), 0), 0))
+    && REG_P (XEXP (XEXP (SET_SRC (curr_set), 0), 1))
+    && REG_P (SET_DEST (prev_set))
+	  && (REGNO (XEXP (XEXP (SET_SRC (curr_set), 0), 0)) == REGNO (SET_DEST (prev_set))
+     || REGNO (XEXP (XEXP (SET_SRC (curr_set), 0), 1)) == REGNO (SET_DEST (prev_set)))
+    && (GET_CODE (SET_SRC (prev_set)) == LO_SUM
+	   || (GET_CODE (SET_SRC (prev_set)) == PLUS
+      && SMALL_OPERAND (INTVAL (XEXP (SET_SRC (prev_set), 1)))))
+    && INTVAL (XEXP (SET_SRC (prev_set), 0)) == 0)
+  return true;
+    }
+
+  if (simple_sets_p && riscv_fusion_enabled_p (RISCV_FUSE_ADDI_JALR))
+    {
+      /* We are trying to match the following:
+	   prev (addi) == (set (reg:DI rD)
+			        (plus:DI (reg:DI rS1) (const_int IMM12)))
+	   curr (jalr) == (set (reg:DI X0)
+              (call: (mem:DI (reg:DI X1)))) */
+
+      if (GET_CODE (SET_SRC (curr_set)) == CALL
+    && MEM_P (XEXP (SET_SRC (curr_set), 0))
+    && INTVAL (SET_DEST (curr_set)) == 0
+	  && INTVAL (XEXP (XEXP (SET_SRC (curr_set), 0), 0)) == 1
+    && (GET_CODE (SET_SRC (prev_set)) == LO_SUM
+	   || (GET_CODE (SET_SRC (prev_set)) == PLUS
+      && SMALL_OPERAND (INTVAL (XEXP (SET_SRC (prev_set), 1))))))
+  return true;
+    }
+
   return false;
 }
 
-- 
2.25.1

