Upstream-Status: Inappropriate [case specific]
diff --git a/Makefile.am b/Makefile.am
index 13a06ab..6076684 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -168,7 +168,7 @@ CHECK_DIR_BIN = $(if $(CHECK_DIR),$(CHECK_DIR)/hmaccalc,$(bindir))
 
 install-exec-hook:
 	$(MKDIR_P) -p $(DESTDIR)/$(pkglibexecdir)
-	$(foreach link, $(hasher_links), $(LN) -sf $(DESTDIR)/$(bindir)/kcapi-hasher $(DESTDIR)/$(pkglibexecdir)/$(link);)
+
 if HAVE_OPENSSL
 	$(MKDIR_P) $(DESTDIR)$(CHECK_DIR_BIN)
 	cd $(DESTDIR)$(bindir) && $(CHECKSUM_CMD) kcapi-hasher > $(DESTDIR)$(CHECK_DIR_BIN)/$(CHECK_PREFIX)kcapi-hasher.$(CHECK_SUFFIX)
