From 63a44e5923c859e99d3a8799fa8132b49a135241 Mon Sep 17 00:00:00 2001
From: Jim Wilson <jimw@sifive.com>
Date: Tue, 17 Nov 2020 19:30:08 -0800
Subject: [PATCH 03/48] Fix riscv sim build on Mac.  elf-bfd.h now requires
 string.h for strncmp.

---
 sim/riscv/interp.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sim/riscv/interp.c b/sim/riscv/interp.c
index f28c841cfc..5557d5bb81 100644
--- a/sim/riscv/interp.c
+++ b/sim/riscv/interp.c
@@ -27,6 +27,9 @@
 
 #include "config.h"
 
+/* elf-bfd.h requires string.h for strncmp.  */
+#include <string.h>
+
 #include "libiberty.h"
 #include "bfd.h"
 #include "elf-bfd.h"
-- 
2.33.0

