From 9b95cdf2e388df25849206c0eab086f63b12c3e9 Mon Sep 17 00:00:00 2001
From: Nelson Chu <nelson.chu@sifive.com>
Date: Mon, 6 Jul 2020 14:46:21 +0800
Subject: [PATCH 16/48] Update the vector spec version tag (Version
 1.0-draft-20200702).

---
 include/opcode/riscv-opc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/opcode/riscv-opc.h b/include/opcode/riscv-opc.h
index 4e4ecc91d0..71eb770f5a 100644
--- a/include/opcode/riscv-opc.h
+++ b/include/opcode/riscv-opc.h
@@ -549,7 +549,7 @@
 #define MASK_C_SDSP  0xe003
 
 /* RVV */
-/* Version 1.0-draft-20200516.  */
+/* Version 1.0-draft-20200702.  */
 
 #define MATCH_VSETVL  0x80007057
 #define MASK_VSETVL   0xfe00707f
-- 
2.33.0

