From 40f63189157ba812686d572b8ec5e177fd40ed10 Mon Sep 17 00:00:00 2001
From: Kito Cheng <kito.cheng@sifive.com>
Date: Thu, 20 Aug 2020 14:25:29 +0800
Subject: [PATCH 30/48] Set default version of ZFH to 0.1

---
 opcodes/riscv-opc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/opcodes/riscv-opc.c b/opcodes/riscv-opc.c
index 89593ef979..af3d787fe6 100644
--- a/opcodes/riscv-opc.c
+++ b/opcodes/riscv-opc.c
@@ -2397,6 +2397,8 @@ const struct riscv_ext_version riscv_ext_version_table[] =
 {"zifencei", ISA_SPEC_CLASS_20191213, 2, 0},
 {"zifencei", ISA_SPEC_CLASS_20190608, 2, 0},
 
+{"zfh", ISA_SPEC_CLASS_NONE, 0, 1},
+
 {"zvamo",   ISA_SPEC_CLASS_NONE, 1, 0},
 {"zvediv",  ISA_SPEC_CLASS_NONE, 1, 0},
 {"zvlsseg", ISA_SPEC_CLASS_NONE, 1, 0},
-- 
2.33.0

