From 798531e05015ae09ddf9a71d6e6abff7df4f89c5 Mon Sep 17 00:00:00 2001
From: "max.ma" <max.ma@starfivetech.com>
Date: Fri, 15 Oct 2021 01:23:18 -0700
Subject: [PATCH 05/11] fix incorrect RCLASS_MAX

---
 gas/config/tc-riscv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gas/config/tc-riscv.c b/gas/config/tc-riscv.c
index cb8d8ef79e..3f6f7848b9 100644
--- a/gas/config/tc-riscv.c
+++ b/gas/config/tc-riscv.c
@@ -829,9 +829,9 @@ enum reg_class
 {
   RCLASS_GPR,
   RCLASS_FPR,
-  RCLASS_MAX,
   RCLASS_VECR,
   RCLASS_VECM,
+  RCLASS_MAX,
   RCLASS_CSR
 };
 
-- 
2.33.1

