From 0f7de0c56b144b5ea2c76289766e2ffd9dc212cc Mon Sep 17 00:00:00 2001
From: LevyHsu <admin@levyhsu.com>
Date: Thu, 10 Jun 2021 16:11:19 +0800
Subject: [PATCH 10/28] [RISCV] Coding bug Fix for FSRIW

---
 include/opcode/riscv-opc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/opcode/riscv-opc.h b/include/opcode/riscv-opc.h
index 5bc9345178..3a86dfc17d 100644
--- a/include/opcode/riscv-opc.h
+++ b/include/opcode/riscv-opc.h
@@ -659,7 +659,7 @@
 #define MASK_FSLW  0x600707f
 #define MATCH_FSRW 0x400503B
 #define MASK_FSRW  0x600707f
-#define MATCH_FSRIW 0x400301b
+#define MATCH_FSRIW 0x400501B
 #define MASK_FSRIW  0x600707f
 #define MATCH_SHFLW 0x800103b
 #define MASK_SHFLW  0xfe00707f
-- 
2.33.0

