From 04d14996c5141373a8bd4402c2dd3625e0ce2bfd Mon Sep 17 00:00:00 2001
From: "max.ma" <max.ma@starfivetech.com>
Date: Tue, 2 Nov 2021 20:23:54 -0700
Subject: [PATCH 11/11] enable l2 cache prefetch

---
 gas/config/tc-riscv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gas/config/tc-riscv.c b/gas/config/tc-riscv.c
index 3df6b6c0f6..d473743019 100644
--- a/gas/config/tc-riscv.c
+++ b/gas/config/tc-riscv.c
@@ -1561,7 +1561,7 @@ check_absolute_expr (struct riscv_cl_insn *ip, expressionS *ex,
 
 static bfd_boolean check_pref_type(unsigned long type)
 {
-  if (type != 0 && type != 1 && type != 3 && type != 8)
+  if (type != 0 && type != 1 && type != 2 && type != 3 && type != 8)
     as_bad (_("Improper pref type (%lu)"), type);
 }
 
-- 
2.33.1

