From cc69150cd40ddd6be172341cd9d19960b99e63db Mon Sep 17 00:00:00 2001
From: LevyHsu <admin@levyhsu.com>
Date: Thu, 17 Jun 2021 13:57:47 +0800
Subject: [PATCH 12/28] update bitmap version number from 0.93 to 0.94

---
 opcodes/riscv-opc.c | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/opcodes/riscv-opc.c b/opcodes/riscv-opc.c
index a01a5c9202..230bbdf11e 100644
--- a/opcodes/riscv-opc.c
+++ b/opcodes/riscv-opc.c
@@ -1980,17 +1980,17 @@ const struct riscv_ext_version riscv_ext_version_table[] =
 
 {"zihintpause", ISA_SPEC_CLASS_DRAFT, 1, 0},
 
-{"b",     ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbb",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zba",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbc",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbe",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbf",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbm",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbp",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbr",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbs",   ISA_SPEC_CLASS_DRAFT, 0, 93},
-{"zbt",   ISA_SPEC_CLASS_DRAFT, 0, 93},
+{"b",     ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbb",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zba",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbc",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbe",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbf",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbm",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbp",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbr",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbs",   ISA_SPEC_CLASS_DRAFT, 0, 94},
+{"zbt",   ISA_SPEC_CLASS_DRAFT, 0, 94},
 
 {"zvamo",   ISA_SPEC_CLASS_NONE, 1, 0},
 {"zvlsseg", ISA_SPEC_CLASS_NONE, 1, 0},
-- 
2.33.0

