From 86c531ca0575841766e539f556436275ec782fe7 Mon Sep 17 00:00:00 2001
From: "max.ma" <max.ma@starfivetech.com>
Date: Tue, 22 Jun 2021 00:40:19 -0700
Subject: [PATCH 14/28] resolve crash issue when objdump encounter unrecognized
 v instruction

---
 opcodes/riscv-opc.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/opcodes/riscv-opc.c b/opcodes/riscv-opc.c
index 230bbdf11e..13ed6500ac 100644
--- a/opcodes/riscv-opc.c
+++ b/opcodes/riscv-opc.c
@@ -553,12 +553,12 @@ match_vd_neq_vs1_neq_vs2_neq_vm (const struct riscv_opcode *op,
   if (!constraints || error == NULL)
     return match_opcode (op, insn);
 
-  if (vs1 == vd)
-    *error = "illegal operands vd cannot overlap vs1";
-  else if (vs2 == vd)
-    *error = "illegal operands vd cannot overlap vs2";
-  else if (!vm && vm == vd)
-    *error = "illegal operands vd cannot overlap vm";
+  if (vs1 == vd) return 0;
+    //*error = "illegal operands vd cannot overlap vs1";
+  else if (vs2 == vd) return 0;
+    //*error = "illegal operands vd cannot overlap vs2";
+  else if (!vm && vm == vd) return 0;
+    //*error = "illegal operands vd cannot overlap vm";
   else
     return match_opcode (op, insn);
   return 0;
-- 
2.33.0

