From 11fdd0a46602d7c6f21801772eccc54ebafbba43 Mon Sep 17 00:00:00 2001
From: LevyHsu <admin@levyhsu.com>
Date: Thu, 24 Jun 2021 14:59:31 +0800
Subject: [PATCH 17/28] Fix coding for Bcompressw

---
 include/opcode/riscv-opc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/opcode/riscv-opc.h b/include/opcode/riscv-opc.h
index 3a86dfc17d..71962e19f0 100644
--- a/include/opcode/riscv-opc.h
+++ b/include/opcode/riscv-opc.h
@@ -665,7 +665,7 @@
 #define MASK_SHFLW  0xfe00707f
 #define MATCH_UNSHFLW 0x800503b
 #define MASK_UNSHFLW  0xfe00707f
-#define MATCH_BCOMPRESSW 0x4800603b
+#define MATCH_BCOMPRESSW 0x800603B
 #define MASK_BCOMPRESSW  0xfe00707f
 #define MATCH_BDECOMPRESSW 0x4800603b
 #define MASK_BDECOMPRESSW  0xfe00707f
-- 
2.33.0

