From ac21c6c5a0ac15f33da81820b03ae493ad1e1e8b Mon Sep 17 00:00:00 2001
From: LevyHsu <admin@levyhsu.com>
Date: Thu, 24 Jun 2021 15:08:51 +0800
Subject: [PATCH 18/28] Fix zip2.b testcase

---
 gas/testsuite/gas/riscv/b-ext-64.d | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gas/testsuite/gas/riscv/b-ext-64.d b/gas/testsuite/gas/riscv/b-ext-64.d
index 8a1daf2766..094ee2d554 100755
--- a/gas/testsuite/gas/riscv/b-ext-64.d
+++ b/gas/testsuite/gas/riscv/b-ext-64.d
@@ -60,7 +60,7 @@ Disassembly of section .text:
 [ 	]+c4:[ 	]+2825d53b[ 	]+gorcw[ 	]+a0,a1,0x2
 [ 	]+c8:[ 	]+2825d51b[ 	]+gorciw[ 	]+a0,a1,0x2
 [ 	]+cc:[ 	]+08c59533[ 	]+shfl[ 	]+a0,a1,a2
-[ 	]+d0:[ 	]+08259513[ 	]+shfli[ 	]+a0,a1,0x2
+[ 	]+d0:[ 	]+08259513[ 	]+zip2.b[ 	]+a0,a1
 [ 	]+d4:[ 	]+0825953b[ 	]+shflw[ 	]+a0,a1,0x2
 [ 	]+d8:[ 	]+08c5d533[ 	]+unshfl[ 	]+a0,a1,a2
 [ 	]+dc:[ 	]+0825d513[ 	]+unshfli[ 	]+a0,a1,0x2
-- 
2.33.0

