From 09a4ef2adddc0478e1ac752c42ecef181dd32bc5 Mon Sep 17 00:00:00 2001
From: LevyHsu <admin@levyhsu.com>
Date: Thu, 24 Jun 2021 15:57:47 +0800
Subject: [PATCH 21/28] Fix testcase for zip2.b and rev2.n

---
 gas/testsuite/gas/riscv/b-ext-64.d | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gas/testsuite/gas/riscv/b-ext-64.d b/gas/testsuite/gas/riscv/b-ext-64.d
index 10aaf70c8a..e27787a3f5 100755
--- a/gas/testsuite/gas/riscv/b-ext-64.d
+++ b/gas/testsuite/gas/riscv/b-ext-64.d
@@ -52,7 +52,7 @@ Disassembly of section .text:
 [ 	]+a4:[ 	]+0825c53b[ 	]+packw[ 	]+a0,a1,0x2
 [ 	]+a8:[ 	]+4825c53b[ 	]+packuw[ 	]+a0,a1,0x2
 [ 	]+ac:[ 	]+68c5d533[ 	]+grev[ 	]+a0,a1,a2
-[ 	]+b0:[ 	]+6825d513[ 	]+rev2.n[ 	]+a0,a1
+[ 	]+b0:[ 	]+6825d513[ 	]+rev2.n[ 	]+a0,a1,0x2
 [ 	]+b4:[ 	]+6825d53b[ 	]+grevw[ 	]+a0,a1,0x2
 [ 	]+b8:[ 	]+6825d51b[ 	]+greviw[ 	]+a0,a1,0x2
 [ 	]+bc:[ 	]+28c5d533[ 	]+gorc[ 	]+a0,a1,a2
@@ -63,7 +63,7 @@ Disassembly of section .text:
 [ 	]+d0:[ 	]+08259513[ 	]+zip2.b[ 	]+a0,a1
 [ 	]+d4:[ 	]+0825953b[ 	]+shflw[ 	]+a0,a1,0x2
 [ 	]+d8:[ 	]+08c5d533[ 	]+unshfl[ 	]+a0,a1,a2
-[ 	]+dc:[ 	]+0825d513[ 	]+unshfli[ 	]+a0,a1,0x2
+[ 	]+dc:[ 	]+0825d513[ 	]+unzip2.b[ 	]+a0,a1,0x2
 [ 	]+e0:[ 	]+0825d53b[ 	]+unshflw[ 	]+a0,a1,0x2
 [ 	]+e4:[ 	]+28c5a533[ 	]+xperm.n[ 	]+a0,a1,a2
 [ 	]+e8:[ 	]+28c5c533[ 	]+xperm.b[ 	]+a0,a1,a2
-- 
2.33.0

